### PR TITLE
BCF 3.0: Add test case for single visible space

### PR DESCRIPTION
**Pull request by mattikannala**
BCF 3.0: Add test case for single visible space

### CODE REVIEW

**Review by cschladetsch — COMMENTED** [review]
Why is there extra vertical whitespace around the indented bullet point? Should the indented bullet point end with a period?

**Comment by mattikannala** [comment]
> Why is there extra vertical whitespace around the indented bullet point? Should the indented bullet point end with a period?

Don't know if it is now better. I tried to fix it.

**Review by ykulbak — APPROVED** [review]
Reviewed and tested on Aconex

**Comment by leosobral** [comment]
Imports OK in Bimsync, but 3D viewer looks quite different with models loaded

**Comment by mattikannala** [comment]
Renamed to lower "case single visible space.bcf"

File: Test Cases/v3.0/Visualization/Single visible space/unzipped/bcf.version [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Version xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" VersionId="3.0" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/buildingSMART/BCF-XML/release_3_0/Schemas/version.xsd">
+    <DetailedVersion>3.0</DetailedVersion>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/Single visible space/unzipped/bcf.version:3]: Remove detailed version

File: Test Cases/v3.0/Visualization/Single visible space/unzipped/a3befb7d-8395-4e52-bbca-382fc4198217/markup.bcf [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+			</File>
+		</Files>
+    </Header>
+    <Topic Guid="a3befb7d-8395-4e52-bbca-382fc4198217" TopicType="Error" TopicStatus="Open">
+        <ReferenceLinks/>
+        <Title>Single visible space</Title>
+        <Index>0</Index>
+        <Labels/>
+        <CreationDate>2017-05-23T11:34:51+03:00</CreationDate>
+        <CreationAuthor>support@solibri.com</CreationAuthor>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/Single visible space/unzipped/a3befb7d-8395-4e52-bbca-382fc4198217/markup.bcf:21]: Maybe use a generic email here.

**Review by pasi-paasiala — CHANGES_REQUESTED** [review]
-1

detailed version must go.

File: Test Cases/v3.0/Visualization/Single visible space/unzipped/bcf.version (hunk 1 above)
Comment (mattikannala) [reply Test Cases/v3.0/Visualization/Single visible space/unzipped/bcf.version:3]: Removed

File: Test Cases/v3.0/Visualization/Single visible space/unzipped/a3befb7d-8395-4e52-bbca-382fc4198217/markup.bcf (hunk 2 above)
Comment (mattikannala) [reply Test Cases/v3.0/Visualization/Single visible space/unzipped/a3befb7d-8395-4e52-bbca-382fc4198217/markup.bcf:21]: Changed to test@bcfxmltest.com

**Comment by mattikannala** [comment]
added extensions.xml

**Comment by mattikannala** [comment]
> -1
> 
> detailed version must go.

It is not there anymore.

**Review by pasi-paasiala — APPROVED** [review]
+2

**Pull request merged**